src/battle/anim/draw.c

```diff
@@ -59,7 +59,6 @@ void sub_80D0C88(u8 taskId)
     task->func = sub_80D0D68;
 }
 
-#ifdef NONMATCHING
 void sub_80D0D68(u8 taskId)
```

Comment:
Why were we unable to match this before?

Reply:
Couldn't tell you.